0005-Flatpak-specific-appdata-changes.patch

```diff
@@ -26,16 +22,27 @@ index 47e7f00..7c2685f 100644
   <metadata_license>FSFAP</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>ClamTk</name>
-@@ -99,6 +99,9 @@
+@@ -79,11 +79,7 @@
+  <screenshots>
+      <screenshot type="default">
+          <caption>The initial screen</caption>
+-         <image>https://davem.fedorapeople.org/images/clamtk-1-screenshot.png</image>
+-     </screenshot>
+-     <screenshot>
+-         <caption>Program settings dialog</caption>
+-         <image>https://davem.fedorapeople.org/images/clamtk-2-screenshot.png</image>
++         <image>https://raw.githubusercontent.com/bbhtt/com.gitlab.davem.ClamTk/release-6.15/screenshot.png</image>
```

Comment:
This url needs to updated after merging. Locally appstream doesn't like the original width and height of https://davem.fedorapeople.org/images/clamtk-{1,2}-screenshot.png and the one in this repository has too much height >900px.

Reply:
Not sure why the screenshots need any changes from the upstream ones?

Reply:
appstream complained locally about the size. Not sure if Flathub's patched one will accept.

Reply:
```
appstream-util validate com.github.davetheunsub.clamtk.appdata.xml 

com.github.davetheunsub.clamtk.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• attribute-invalid     : <screenshot> width (504) too small [https://davem.fedorapeople.org/images/clamtk-1-screenshot.png] minimum is 624px
• attribute-invalid     : <screenshot> width (504) too small [https://davem.fedorapeople.org/images/clamtk-2-screenshot.png] minimum is 624px
• style-invalid         : <p> is too short [Bug fixes]
Validation of files failed
```